crates/ty_python_semantic/src/module_resolver/resolver.rs

```diff
@@ -796,7 +809,7 @@ fn resolve_name_in_search_path(
     context: &ResolverContext,
     name: &RelaxedModuleName,
     search_path: &SearchPath,
-) -> Result<(PackageKind, ResolvedName), PackageKind> {
+) -> Result<(PackageKind, PyTyped, ResolvedName), (PackageKind, PyTyped)> {
```

Comment:
This is really gross and I considered factoring a proper type *but* literally only one function deals with this and immediately unpacks these so... eh?